PortfolioApp/UsersScreen/Users/UsersTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+final class UsersTableViewController: UITableViewController {
+    public typealias ReusableType = UsersTableViewController
+    @IBOutlet private weak var usersTableView: UITableView!
+    var presenter = UsersPresenter()
+    var models: [User] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter.view = self
```

Comment:
You already did this in Composer.
`final class UserComposer {
    static func build() -> UIViewController {
        let presenter = UsersPresenter()
        let usersVC = UsersTableViewController(nib: R.nib.usersTableViewController)
        usersVC.presenter = presenter  // HERE!!
        presenter.view = usersVC
        return usersVC
    }
}`